bundle/README.md

```diff
@@ -71,6 +71,36 @@ For a list of available configuration for the `values` field, refer to [Istio's
 - [CNI parameters](https://artifacthub.io/packages/helm/istio-official/cni?modal=values)
 - [ZTunnel parameters](https://artifacthub.io/packages/helm/istio-official/ztunnel?modal=values)
 
+## Installing the istioctl tool
+
+The istioctl tool is a configuration command line utility that allows service operators to debug and diagnose Istio service mesh deployments.
```

Comment:
surround `istioctl` with back quotes everywhere

Reply:
Done, except in the title, which is the house style.